mobile.css

```diff
@@ -923,3 +934,4 @@ body {
     left: 12px
 }
 }
+/* Off the media query */
```

Comment:
- Great Job implementing the desktop version of your portfolio. 💪🏽 
-  I'd like to suggest that you fix your porftolio on screensize 375px. Just as the previous reviewer suggested, kindly remove the `max-width 360px` so that the problem can be fixed. 🌸

Reply:
:orange_circle: [OPTIONAL] You did a great move since the first review. However there is still room for improvement :arrow_down:  Kindly fix these Design issues

|YOUR DESIGN|FIGMA DESIGN|
|:--------------:|:-------------|
|![Screenshot from 2023-01-27 10-48-59](https://user-images.githubusercontent.com/26736582/215046405-f4fb2bbe-7aac-4c2c-9174-2516c7d30ee2.png)|![Screenshot from 2023-01-27 10-59-54](https://user-images.githubusercontent.com/26736582/215047623-c3eed718-2c5e-4cdf-9f0d-db8c2158479f.png)
|![Screenshot from 2023-01-27 10-46-14](https://user-images.githubusercontent.com/26736582/215046411-f8334e8d-8151-4197-a719-1fe672291b4e.png)|![Screenshot from 2023-01-27 11-01-29](https://user-images.githubusercontent.com/26736582/215047912-101976d2-498b-4ccb-ac86-3d9cca572451.png)|
|![Screenshot from 2023-01-27 10-46-00](https://user-images.githubusercontent.com/26736582/215046419-15ca5e1b-06f4-4aaa-b0c7-e3585f712961.png)|![Screenshot from 2023-01-27 11-01-29](https://user-images.githubusercontent.com/26736582/215047912-101976d2-498b-4ccb-ac86-3d9cca572451.png)|
|![Screenshot from 2023-01-27 10-45-36](https://user-images.githubusercontent.com/26736582/215046422-92ad2675-4282-4854-93e2-29811da932df.png)|![Screenshot from 2023-01-27 11-02-37](https://user-images.githubusercontent.com/26736582/215048089-1585affe-3414-4b98-ac40-651e327b4b01.png)|
|![Screenshot from 2023-01-27 10-40-47](https://user-images.githubusercontent.com/26736582/215046425-80c18a62-334b-46c3-b59b-4f34106c6e03.png)|![Screenshot from 2023-01-27 10-40-47](https://user-images.githubusercontent.com/26736582/215048310-85394364-2678-47da-9973-a24b3ac75a2d.png)|